cmd/otk/osbuild-resolve-ostree-commit/main.go

```diff
@@ -49,7 +49,10 @@ func run(r io.Reader, w io.Writer) error {
 		return err
 	}
 
-	sourceSpec := ostree.SourceSpec(inputTree.Tree)
+	sourceSpec := ostree.SourceSpec{}
+	sourceSpec.URL = inputTree.Tree.URL
+	sourceSpec.Ref = inputTree.Tree.Ref
+	sourceSpec.RHSM = inputTree.Tree.RHSM
```

Comment:
```suggestion
sourceSpec := ostree.SourceSpec{
	URL: inputTree.Tree.URL,
	Ref: inputTree.Tree.Ref,
	RHSM: inputTree.Tree.RHSM,
}
```

(again, nitpick)

Reply:
Is absolutely nicer way of writing that and I would do that, but this is like 5th version of this patch so I kept moving things around and this was the reasult. Fixed both.